money-http-client/src/main/scala/com/comcast/money/http/client/TraceFriendlyHttpClient.scala

```diff
@@ -122,12 +122,12 @@ class TraceFriendlyHttpClient(wrapee: HttpClient) extends HttpClient with java.i
   }
 
   override def close(): Unit = {
-    if (wrapee.isInstanceOf[CloseableHttpClient])
-      wrapee.asInstanceOf[CloseableHttpClient].close()
-    else if (wrapee.isInstanceOf[Closeable])
-      wrapee.asInstanceOf[Closeable].close()
-    else if (wrapee.isInstanceOf[AutoCloseable])
-      wrapee.asInstanceOf[AutoCloseable].close()
+    wrapee match {
```

Comment:
nice cleanup.  I think money in general needs a refresh / update because there is likely lots of opportunity for code cleanup